### ISSUE TITLE
Strawberry sometimes crashes when putting one or multiple songs into a playlist (linux/win10)

### ISSUE THREAD

**Issue by Leetsch2002** [strawberrymusicplayer/strawberry#701]
**Describe the bug**
When I try to add a song from my collection to a playlist by dragging it to the place I want to have it in the playlist, strawberry sometimes just crashes. The song is not in the playlist when I restart strawberry. When I try again, it mostly works. I can't find a scheme of which files work and which won't, it seems random. 

**To Reproduce**
For me, the bug is reproducable by adding songs per drag & drop to playlists. I assume that it is generally possible to reproduce and that I'm not the only one having this problem, because I had it on my last installation (Linux Mint) as well as on my Windows 10 installation

**Expected behavior**
When I drag a song in a playlist I expect strawberry to put it there and not to crash.

**System Information:**
 - Operating system: Garuda Linux (arch-based, KDE Plasma)
 - Strawberry Version: 0.9.3

**Comment by jonaski** [comment]
I have never seen this crash, and no-one else has reported it.
A corrupt database can cause this:
https://github.com/strawberrymusicplayer/strawberry/wiki/Database-integrity-check

If that doesn't solve it, you need to do a backtrace. Make sure you paste the backtrace within `code`.
https://github.com/strawberrymusicplayer/strawberry/wiki/Backtrace

**Comment by Leetsch2002** [comment]
I tried to reproduce the issue, but it does not occur anymore. I don't understand why, I had this issue for a long time on different operating systems. I also don't think that I updated Strawberry. I did the integrity check of the database, and it said the database is fine. I don't think it's useful to post the backtrace if nothing crashed.
If I have this issue again I'll do a backtrace and post it here, but until then, I can't really explain why it doesn't crash anymore (I'm happy it doesn't crash anymore, but I don't understand why)
